webservice/soap/classes/class.ilSoapUserAdministration.php

```diff
@@ -99,6 +99,71 @@ function loginLDAP($client, $username, $password)
 		return $this->login($client, $username, $password);
 	}
 
+	/**
+	 * login as user for studip mode
+	 * @param string $sid
+	 * @param int $user_id
+	 * @return string $sid
+	 */
+	public function loginStudipUser($sid, $user_id)
```

Comment:
According to the ILIAS Jour Fixe: Please mark this method/service as @deprecated and add a short note, because this service will be removed with ILIAS 5.3.x

- http://www.ilias.de/mantis/view.php?id=18992


-